src/browser-polyfill.js

```diff
@@ -13,7 +13,7 @@ if (typeof browser === "undefined") {
   // never actually need to be called, this allows the polyfill to be included
   // in Firefox nearly for free.
   const wrapAPIs = () => {
-    const apiMetadata = {/* include("api-metadata.json") */};
+    const apiMetadata = require("../api-metadata.json");
```

Comment:
> My guess is that you are preferring the require here so that the github repo can be used as an npm dependencies with browserify/webpack without the need for a build step.

That's a side benefit, yes, but my goal this time was actually to make it so that this package can be `import`ed or `require()`ed by other projects without including UMD boilerplate, while build process still produces a UMD module for use in a script tag.

> But I'm a bit concerned that by looking at the sources (without looking at the building process) this require can be misinterpreted

I definitely agree here, and even felt a little dirty writing the `grunt-replace` rule for the `require()` syntax (especially since it has to strip out the leading `../` in the path) 😝 

However, that I think about it, there is a nice incidental effect of having it this way: It makes it much harder for people to accidentally use the source version of the polyfill (which happened in https://github.com/mozilla/webextension-polyfill/issues/5)

Ultimately, though, I'm not really devoted to this syntax, and would happily switch back to `{/* include("...") */}` if that's what the team would prefer.